components/datasource-selector/partials/datasource-selector.html

```diff
@@ -37,30 +37,14 @@
                         <div *ngIf="endpoint.type == 'micka'">
                             <hs-micka-filters micka-dataset-config="{url: endpoint.url}"></hs-micka-filters>
                         </div>
-                        <hs-layman-current-user endpoint="endpoint"></hs-layman-current-user>
+                        <hs-layman-current-user [endpoint]="endpoint"></hs-layman-current-user>
```

Comment:
This one doesnt seem work. Updated testing app to this branch and layman-current user is missing from datasources panel
https://hub.lesprojekt.cz/hsl-ng/hsl-ng-2.x/apps/full

Reply:
@FilipLeitner  Probably because in this branch its not rewritten yet. I did it in Composition ng9 pullrequests commit: https://github.com/hslayers/hslayers-ng/pull/1146/commits/bd8337cbacb542795af50d4248ec9e9f46c1fbaa

Reply:
@raitisbe It's not rewritten in this branch, but it should be upgraded. I made several mistakes with the upgrade, have corrected some, but there is still something I miss... 🙁